lib/ansible/modules/windows/win_acl.ps1

```diff
@@ -161,11 +161,12 @@ Try {
     # Workaround to handle special use case 'APPLICATION PACKAGE AUTHORITY\ALL APPLICATION PACKAGES' and
     # 'APPLICATION PACKAGE AUTHORITY\ALL RESTRICTED APPLICATION PACKAGES'- can't translate fully qualified name (win32 API bug/oddity)
     # 'ALL APPLICATION PACKAGES' exists only on Win2k12 and Win2k16 and 'ALL RESTRICTED APPLICATION PACKAGES' exists only in Win2k16
-    $specialIdRefs = "ALL APPLICATION PACKAGES","ALL RESTRICTED APPLICATION PACKAGES"
+    $specialIdRefPrefixes = "APPLICATION PACKAGE AUTHORITY"
     ForEach($rule in $objACL.Access){
+        $idRefShortPrefix = ($rule.IdentityReference.Value).split('\')[0]
         $idRefShortValue = ($rule.IdentityReference.Value).split('\')[-1]
 
-        if ( $idRefShortValue -in $specialIdRefs ) {
+        if ( $idRefShortPrefix -in $specialIdRefPrefixes ) {
```

Comment:
This is probably a really tiny nitpick, but would 
`if ( $idRefShortPrefix -eq $specialIdRefPrefixes ) { `
be clearer here, since $specialIdRefPrefixes is just a string not an Array?
I think -in and -eq behave the same in this case but I think -eq would make it clearer what the intention is.

Reply:
I prefer changing `specialIdRefPrefixes` to be an array or future additions